selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -87,10 +88,11 @@ void Networking::attemptInitialization() {
   )");
   main_layout->setCurrentWidget(wifiScreen);
   ui_setup_complete = true;
+  wifi->requestScan();
```

Comment:
switch the order here so you can use this in refresh()

Reply:
It was non-blocking so the order wouldn't have mattered